eng/Version.Details.xml

```diff
@@ -1,16 +1,16 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.4.24253.11">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.4.24257.5">
```

Comment:
We are testing this one here as well: https://github.com/xamarin/xamarin-android/pull/8932